frontend/src/components/Layout/Header/header.scss

```diff
@@ -0,0 +1,7 @@
+@import "../../../styles/components/variables";
+
+#BC-Header {

```

Comment:
What naming conventions are we going to use BCHeader BC-Header bc-header... I think ive seen different conventions.  We should figure this out sooner than later...

Reply:
I was thinking similar to PHP so I agree it should be changed to bc-header